File: Spring/Chapter7/MyBatisSpringJavaInit/src/main/java/pers/manyToMany/entity/Product.java
package pers.manyToMany.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import pers.oneToMany.entity.Orders;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Product {
    private Integer id;
    private String name;
    private Double price;
    private List<Orders> orders;
}

File: Spring/Chapter7/MyBatisSpringKotlinInit/src/main/kotlin/pers/manyToMany/entity/Product.kt
package pers.manyToMany.entity

import pers.oneToMany.entity.Orders

class Product {
    var id = 0
    var name = ""
    var price = 0.0
    var orders = mutableListOf<Orders>()
    override fun toString() = "Product(id:$id,name:$name,price:$price,orders:$orders)"
}